packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -127,11 +126,12 @@ var ReactDOMEventListener = {
     if (!element) {
       return null;
     }
-    return EventListener.listen(
-      element,
-      handlerBaseName,
-      ReactDOMEventListener.dispatchEvent.bind(null, topLevelType),
-    );
+    var callback = ReactDOMEventListener.dispatchEvent.bind(null, topLevelType);
+    if (element.addEventListener) {
+      element.addEventListener(handlerBaseName, callback, false);
+    } else if (element.attachEvent) {
```

Comment:
Is it possible that `element.attachEvent` is undefined? If it is never the case, maybe leave this as an else?

Reply:
Could be in some strange environment or if this isn't a real element some how. Probably shouldn't happen but since this is the logic we're currently shipping in 16.x, it's safer to keep it.

Reply:
Let's just drop this path?